### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by waam**
added fr.js

### CODE REVIEW

File: fr.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+l10n.translations.fr = {
+  logout: 'Déconnexion',
+  error: {
+    a: 'Il y a eu un petit soucis. Merci de',
+    b: 'recharger',
+    c: 'Voici quelques détails techniques :'
```
Comment (dmitry-mukhin) [inline fr.js]: extra space?

File: fr.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    cookieWarning: "Vous avez bloqué les cookies tiers dans votre navigateur. " +
+      "Vous devez autoriser les cookies pour le domaine " +
+      "<code>social.uploadcare.com</code>.",
+   cookieRetry: "J'ai autorisé les cookies et je veux réessayer."
+  },
+  pagination: {
+    loading: 'Chargement...',
+    loadMore: 'Charger plus d\'éléments',
+    failed: {
+      a: 'Le chargement a échoué..',
```
Comment (dmitry-mukhin) [inline fr.js]: not enough dots?

**Comment by dmitry-mukhin** [comment]
we'll fix it.
thanks!

File: fr.js (hunk 1 above)
Comment (waam) [inline fr.js]: Yes. In french, there is a space before two-elements' punctuation marks (? ! : ;). But i should replace it with a non-breakable one.

File: fr.js (hunk 2 above)
Comment (waam) [inline fr.js]: Right. I have to add one.

**Pull request merged**